sns_ik_lib/src/sns_position_ik.cpp

```diff
@@ -119,7 +118,23 @@ int SNSPositionIK::CartToJnt(const KDL::JntArray& joint_seed,
 
     //std::cout << ii << ": Cartesian error: " << L << " m, " << theta << " rad" << std::endl;
 
-    if (lineErr <= linearTolerance && rotErr <= angularTolerance) {
+    // Check stopping tolerances
```

Comment:
This removes the hard coded tolerance values and matches how trac-ik calculates their error stopping condition.